.github/workflows/docker-image-verify.yaml

```diff
@@ -0,0 +1,138 @@
+name: Reusable GCR Vulnerability Docker container scanning
+
+on:
+  workflow_call:
+    inputs:
+      working_directory:
+        required: true
+        type: string
+      dockerfile_path:
+        required: true
+        type: string
+      docker_image_name:
+        required: true
+        type: string
+      docker_image_tag:
+        required: true
+        type: string
+      scan_output_format:
+        required: false
+        type: string
+        default: "table(vulnerability.effectiveSeverity, vulnerability.cvssScore, noteName, vulnerability.packageIssue[0].affectedPackage, vulnerability.packageIssue[0].affectedVersion.name, vulnerability.packageIssue[0].fixedVersion.name)"
+      jira_project:
+        type: string
+        required: true
+      jira_issuetype:
+        type: string
+        required: true
+      jira_jql:
+        type: string
+        required: true
+    secrets:
+      GCP_JSON_KEY:
+        required: true
+
+      JIRA_BASE_URL:
+        required: true
+      JIRA_USER_EMAIL:
+        required: true
+      JIRA_API_TOKEN:
+        required: true
+
+jobs:
+  docker-verify-image:
+    runs-on: [self-hosted, common]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Docker Context for Buildx
+        run: |
+          docker context create builders
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          version: latest
+          endpoint: builders
+
+      - name: Build image
+        working-directory: ${{ inputs.working_directory }}
+        run: |
+          docker build . -f ${{ inputs.dockerfile_path }} -t ${{ inputs.docker_image_name }}:${{ inputs.docker_image_tag }}
```

Comment:
Why do you need to build the image if you scan the images already presented in gcr?

Reply:
1. Because upstream dependencies might change over the time
2. We build images on a) iteration branch deploy  b) right before deploy from master, there was no artifact to verify in between

Reply:
You always have latest tag. Just scan it. There is no need to scan each and every image comes into master because we are not going to fix the vulnerability immediately, it will be ongoing process which is not going to block any deployment

Reply:
still actual